.github/workflows/docs.yaml

```diff
@@ -24,12 +25,15 @@ jobs:
       - name: output
         id: output
         run: |
-          if [ ${{ github.repository_owner }} != 'betaflight' ]; then
-            echo "URL=https://${{ github.repository_owner }}.github.io" >> $GITHUB_OUTPUT
-            echo "BASE_PATH=/${GITHUB_REPOSITORY#*/}/" >> $GITHUB_OUTPUT
-          else
+          if [ ${{ github.repository_owner }} == 'freasy' ]; then
```

Comment:
This will never evaluate as true :)

Reply:
this is so i can test github pages on my own repository